### ISSUE TITLE
PolarDB - SSL In Transit Endpoints

### ISSUE THREAD

**Issue by brianmori** [aliyun/terraform-provider-alicloud#3240]
The SSL setting parameters are missing to configure SSL In Transit in PolarDB for both endpoints

**Comment by tiankai66** [comment]
hello, 
make sure we have a same understanding for this issues:
I want to confirm with you that the SSL setting you mentioned is related to the following apis?
polarDB SSL APIs：
DescribeDBClusterSSL： https://help.aliyun.com/document_detail/185358.html?spm=a2c4g.11186623.2.12.65bb136599zQxT
ModifyDBClusterSSL： https://help.aliyun.com/document_detail/185357.html?spm=a2c4g.11186623.6.1085.70262b71l5gQ3g

**Comment by brianmori** [comment]
Hi,

I do not speak Chinese, the team needs to have PolarDB connections encrypted, I think it is those two APIs.
It would be great to have APIs in English

**Comment by tiankai66** [comment]
hi, 
thanks for your quick feedback. 
hope those links can help you :
https://api.aliyun.com/#/?product=polardb&api=DescribeDBClusterSSL
https://api.aliyun.com/#/?product=polardb&api=ModifyDBClusterSSL

**Comment by brianmori** [comment]
> hi,
> thanks for your quick feedback.
> hope those links can help you :
> https://api.aliyun.com/#/?product=polardb&api=DescribeDBClusterSSL
> https://api.aliyun.com/#/?product=polardb&api=ModifyDBClusterSSL

yes, I think it is these two APIs which allow SSL In-Transit for PolarDB

**Comment by xuzhang3** [comment]
SSL In Transit is now fully supported across all three PolarDB endpoint resources:

- `alicloud_polardb_endpoint` — `ssl_enabled` (v1.121.0+) + `ssl_auto_rotate` (v1.132.0+)
- `alicloud_polardb_cluster_endpoint` — `ssl_enabled` + `ssl_auto_rotate`
- `alicloud_polardb_primary_endpoint` — `ssl_enabled` + `ssl_auto_rotate`

Example:
```terraform
resource "alicloud_polardb_endpoint" "example" {
  db_cluster_id    = alicloud_polardb_cluster.example.id
  endpoint_type    = "Custom"
  ssl_enabled      = "Enable"
  ssl_auto_rotate  = "Enable"
}
```
See docs: https://registry.terraform.io/providers/aliyun/alicloud/latest/docs/resources/polardb_endpoint
